src/OrbitClientGgp/ClientGgp.cpp

```diff
@@ -215,8 +217,14 @@ ErrorMessageOr<void> ClientGgp::LoadModuleAndSymbols() {
   // Load symbols for the module
   const std::string& module_path = main_module_->file_path();
   ORBIT_LOG("Looking for debug info file for %s", module_path);
-  OUTCOME_TRY(auto&& main_executable_debug_file,
+  OUTCOME_TRY(orbit_base::NotFoundOr<std::filesystem::path> && find_result,
               process_client_->FindDebugInfoFile(module_path, {}));
+  if (orbit_base::IsNotFound(find_result)) {
+    return ErrorMessage{
+        absl::StrFormat("Symbols not found: %s", orbit_base::GetNotFoundMessage(find_result))};
+  }
+  ORBIT_CHECK(std::holds_alternative<std::filesystem::path>(find_result));
+  const auto& main_executable_debug_file{std::get<std::filesystem::path>(find_result)};
```

Comment:
Similarly to the previous comment, could we add a dedicated function (basically opposite to `orbit_base::GetNotFoundMessage`) to extract the value instead of using `std::get`? I feel like the fact that `NotFoundOr` uses `std::variant` should not show and just be an implementation detail. What do you think?

Reply:
I agree and introduced `orbit_base::GetFound(...)` an additional nicety is that the template can now be inferred. ptal

Reply:
I'll have another quick look then.